test-values.yaml

```diff
@@ -143,25 +179,9 @@ accessControl:
         ]
       }
 
-jmx:
```

Comment:
Moving this to coordinator is a breaking change. Can we avoid it? At the very least, we should deprecate them first - stop documenting them but still recognize these values in templates.

I wonder if we want to reuse some of these properties for both coordinator and worker, like the image name and ports. I can't think of a use case where different values should be used there.

How about adding a `coordinator` and `worker` sections under `exporter`, that would allow overriding properties from the `exporter` section?

Reply:
I agree the image name and ports should be the same, it just adds entropy.
I like adding `coordinator` and `worker` sections under exporter!